test/Routes/ListRoutesCommandFactoryTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MezzioTest\Tooling\Routes;
+
+use Mezzio\Router\RouteCollector;
+use Mezzio\Tooling\Routes\ListRoutesCommand;
+use Mezzio\Tooling\Routes\ListRoutesCommandFactory;
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Container\ContainerInterface;
+
+class ListRoutesCommandFactoryTest extends TestCase
+{
+    use ProphecyTrait;
```

Comment:
Let's use built-in mocks instead

Reply:
Okay. Sure. Will do.

Reply:
Rule of thumb: do not use `prophecy` anymore. 😅